playbooks/roles/openvpn/tasks/main.yml

```diff
@@ -1,4 +1,15 @@
 ---
+- name: Import the public GPG key from OpenVPN community
+  apt_key: url=https://swupdate.openvpn.net/repos/repo-public.gpg state=present
+
+- name: Adds the OpenVPN community repositroy to sources.list
```

Comment:
There are a few small typos here. The name description should be something like "Add the official OpenVPN Community repository" in order to match the convention from other roles.